packages/react/spec/useAction.spec.tsx

```diff
@@ -805,4 +805,38 @@ describe("useAction", () => {
 
     expect(result.current[0]).toBe(beforeObject);
   });
+
+  test("sends a stubbedActionError event when the action is stubbed", () => {
+    let eventDispatched: CustomEvent | undefined;
+
+    globalThis.addEventListener("gadget:devharness:stubbedActionError", (event) => {
+      eventDispatched = event as CustomEvent;
+    });
+
+    const { result, rerender } = renderHook(
+      () =>
+        useAction({
+          // @ts-expect-error intentionally passing the wrong type
+          type: "stubbedAction",
+          reason: "MissingApiTrigger",
+          dataPath: "fakePath",
+          functionName: "fakeFunction",
+          modelApiIdentifier: "fakeModel",
+          variables: {},
+        }),
```

Comment:
These parameters are unavailable today, so I need to make a fake object to trigger an event. I'll update the tests once we have shipped the change to the Gadget side.